.github/actions/install-boost/action.yml

```diff
@@ -39,7 +39,7 @@ runs:
       if: runner.os == 'macOS'
       shell: bash
       run: |
-        brew install boost@1.81
+        brew install boost
```

Comment:
Unpinning this as it seems brew doesn't keep the previous packages as 1.8 is being updated.